tokio-trace-fmt/src/filter.rs

```diff
@@ -101,8 +101,11 @@ where
     }
 }
 
-impl Filter for EnvFilter {
-    fn callsite_enabled(&self, metadata: &Metadata, _: &span::Context) -> Interest {
+impl<N> Filter<N> for EnvFilter
+where
+    N: for<'a> ::NewVisitor<'a>,
```

Comment:
I think the filter shouldn't care if `N` is a `NewVisitor` or not --- if only the `Context::new_visitor` function had this bound, we could remove it here.